Automattic/wp-calypso
# Tailored Flows: Remove the link in Bio flow

Part of 
- https://github.com/Automattic/dotcom-forge/issues/9333

## Proposed Changes

Removes the code associated to the `/setup/link-in-bio` tailored blow since we no longer use it.

## Why are these changes being made?

To clean up unused code.

## Testing Instructions

- Go to `/setup/link-in-bio`
- Make sure it still redirects to `/start`
- Go to `/setup/link-in-bio-tld`
- Make sure it works as expected

## Pre-merge Checklist
